clap_derive/examples/README.md

```diff
@@ -76,3 +76,7 @@ How to use aliases
 ### [`true` or `false`](true_or_false.rs)
 
 How to express "`"true"` or `"false"` argument.
+
+### [Author, description, and version from `Cargo.toml`](version_from_crate.rs)
```

Comment:
Let's rename this file `from_crate.rs` since it also includes author & description